src/modules/35-connectors/components/ConnectorReferenceField/FormConnectorReferenceField.tsx

```diff
@@ -6,13 +6,13 @@
  */
 
 import React from 'react'
-import { connect, FormikContext } from 'formik'
+import { connect, FormikContextType } from 'formik'
 import { get } from 'lodash-es'
 import { errorCheck } from '@common/utils/formikHelpers'
 import { Scope } from '@common/interfaces/SecretsInterface'
 import { ConnectorReferenceFieldProps, ConnectorReferenceField } from './ConnectorReferenceField'
 export interface FormConnectorFieldProps extends Omit<ConnectorReferenceFieldProps, 'onChange' | 'error'> {
-  formik?: FormikContext<any>
+  formik?: FormikContextType<any>
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
  formik?: FormikContextType<unknown>
```